src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpSyntaxGeneratorInternal.cs

```diff
@@ -239,4 +241,97 @@ public override SyntaxNode UnaryPattern(SyntaxToken operatorToken, SyntaxNode pa
         => SyntaxFactory.UnaryPattern(operatorToken, (PatternSyntax)Parenthesize(pattern));
 
     #endregion
+
+    public override SyntaxNode CastExpression(SyntaxNode type, SyntaxNode expression)
```

Comment:
these are all moves.  From CSharpSyntaxGenerator to CSharpSyntaxGeneratorInternal.